.github/actions/compile-commit/action.yml

```diff
@@ -21,6 +21,8 @@ runs:
         # use tree hashes instead of commit hashes to reuse cache when changes in commits are reorganized without changing the workspace
         key: compile-commit-success-${{ github.event.pull_request.head.repo.full_name }}-${{ steps.repo-hash.outputs.tree-hash }}
     - uses: ./.github/actions/setup
+      with:
+        cleanup-node: true
```

Comment:
why?

Reply:
it was failing due to the lack of space during build. I have no idea why